.gitignore

```diff
@@ -1,6 +1,6 @@
 .DS_Store
 node_modules
-/dist
+# /dist
```

Comment:
Be careful, the `/dist` should not be commited :)

Reply:
I wasn't sure if this repository is still actively maintained, so until that point I'm using my repo to use it as a package with the dist included. I can take it out again for this PR though. Do you merge rights?